Apps/WebClient/src/ClientApp/src/components/modal/noteEdit.vue

```diff
@@ -175,10 +181,8 @@ export default class NoteEditComponent extends Vue {
     }
 
     private onNoteAdded(note: UserNote) {
-        this.eventBus.$emit(
-            EventMessageName.AddedNote,
-            new NoteTimelineEntry(note)
-        );
+        this.clearFilter();
```

Comment:
I don't think we clear the filter when a note is added, just pointing out that this is a behaviour change was not expected, confirm with nino/mandad?!